src/api/collections-api.tsx

```diff
@@ -0,0 +1,14 @@
+import { getConfig, CollectionsApi, ListCollectionsResponse } from "@imtbl/core-sdk";
+
+// Get a list of collections
+// Inspired from Immutable Core SDK documentation
+const GetYourCollections = async (): Promise<ListCollectionsResponse> => {
```

Comment:
better to name GetCollections or listCollections, as they are not user's.

Reply:
Please name function with first letter lower case, upper case uses only for classes